### PR TITLE
Enrollment form + Сhat + Profile + Reports

### PR DESCRIPTION
**Pull request by Squarix**
Made editions according to the previous request comment

### CODE REVIEW

**Comment by PFdev6** [comment]
waiting for))

**Comment by PFdev6** [comment]
👍

File: db/migrate/20190221213300_add_user_id_to_users.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddUserIdToUsers < ActiveRecord::Migration[5.2]
```
Comment (str1nker) [inline db/migrate/20190221213300_add_user_id_to_users.rb]: #Duplicate migration!

**Comment by str1nker** [comment]
Label imposed on value.

default

**Comment by str1nker** [comment]
Time picker doesn't change value

**Comment by str1nker** [comment]
About approve, can admin approve two or more times one users?

**Comment by str1nker** [comment]
On table foundations wrong calculation of students number and column manager empty.

**Comment by str1nker** [comment]
parents dashboard.... just WOW
default

**Comment by PFdev6** [comment]
> On table foundations wrong calculation of students number and column manager empty.
https://github.com/YushPWNZ/schooldogs/tree/fixes_with_admin_dashboard

**Comment by str1nker** [comment]
Bug after add existing child and put on settings.
default

**Comment by PFdev6** [comment]
Thanks, i will fix)

**Comment by str1nker** [comment]
Okey, i just try to check all) Thanks for your job.

**Comment by PFdev6** [comment]
> Time picker doesn't change value

I spend 2 hours on that, but HZ how it is fix

**Comment by str1nker** [comment]
Let's figure it out together)

**Pull request merged**